build.sbt

```diff
@@ -57,6 +57,11 @@ ThisBuild / githubWorkflowBuild :=
       UseRef.Public("coursier", "setup-action", "v1"),
       params = Map("apps" -> "scalafmt:3.8.3")
     ),
+    WorkflowStep.Use(
+      UseRef.Public("actions", "setup-node", "v3"),
+      name = Some("Setup NodeJS v18 LTS"),
+      params = Map("node-version" -> "18")
```

Comment:
```suggestion
      name = Some("Setup NodeJS LTS"),
      params = Map("node-version" -> "22")
```

v18 is already at maintenance phase.
v22 is two years longer, so I think it would be a better choice.


<img width="735" alt="image" src="https://github.com/user-attachments/assets/20709571-2f44-4bd5-8dd8-6481c29d2a1f" />

Reply:
It was a deliberate choice, since our upstream (core library actually), Cats-Effect, is on v18 in their CI. If it's not a thing, we can bump it.